pkg/commands/sso/root.go

```diff
@@ -188,6 +223,55 @@ func (c *RootCommand) identifyProfileAndFlow() (profileName string, flow Profile
 	}
 }
 
+func (c *RootCommand) processCustomer(ar auth.AuthorizationResult) error {
+	debugMode, _ := strconv.ParseBool(c.Globals.Env.DebugMode)
+	apiEndpoint, _ := c.Globals.APIEndpoint()
+	// NOTE: The endpoint is documented but not implemented in go-fastly.
+	data, err := undocumented.Call(undocumented.CallOptions{
+		APIEndpoint: apiEndpoint,
+		HTTPClient:  c.Globals.HTTPClient,
+		HTTPHeaders: []undocumented.HTTPHeader{
+			{
+				Key:   "Accept",
+				Value: "application/json",
+			},
+			{
+				Key:   "User-Agent",
+				Value: useragent.Name,
+			},
+			{
+				Key:   "Fastly-Key",
+				Value: ar.SessionToken,
+			},
+		},
+		Method: http.MethodGet,
+		Path:   "/current_customer",
+		Token:  ar.SessionToken,
+		Debug:  debugMode,
+	})
+	if err != nil {
+		c.Globals.ErrLog.Add(err)
+		return fmt.Errorf("error executing verify API request: %w", err)
+	}
+
+	var response CurrentCustomerResponse
+	if err := json.Unmarshal(data, &response); err != nil {
+		c.Globals.ErrLog.Add(err)
+		return fmt.Errorf("error decoding verify API response: %w", err)
```

Comment:
Nit: the error message is no longer correct:

```suggestion
		return fmt.Errorf("error executing current_customer API request: %w", err)
	}

	var response CurrentCustomerResponse
	if err := json.Unmarshal(data, &response); err != nil {
		c.Globals.ErrLog.Add(err)
		return fmt.Errorf("error decoding current_customer API response: %w", err)
```